plugins/inventory/terraform.py

```diff
@@ -169,6 +183,38 @@ def openstack_host(resource, tfvars=None):
     return name, attrs, groups
 
 
+@parses('aws_instance')
+@calculate_mi_vars
+def aws_host(resource, tfvars=None):
+    name = resource['primary']['attributes']['tags.Name']
+    raw_attrs = resource['primary']['attributes']
+
+    groups = []
+
+    tags = parse_dict(raw_attrs, 'tags')
+    attrs = {
```

Comment:
this should be fleshed out to translate *all* the information that Terraform provides. I can do it pretty quickly if you like, just send me your `.tfstate` file.